bindings/rust/wasmedge-sys/src/instance/memory.rs

```diff
@@ -127,13 +284,34 @@ impl Drop for Memory {
     }
 }
 
+/// Struct of WasmEdge MemType.
+///
+/// A [`MemType`] classifies a [`Memory`] and its size range.
 #[derive(Debug)]
 pub struct MemType {
     pub(crate) ctx: *mut wasmedge::WasmEdge_MemoryTypeContext,
     pub(crate) registered: bool,
 }
 impl MemType {
-    pub fn create(limit: Range<u32>) -> WasmEdgeResult<Self> {
+    /// Create a new [`MemType`] to be associated with the given limit range for the capacity.
+    ///
+    /// # Arguments
+    ///
+    /// - `limit` specifies the linear memory size. The start value of the limit range
+    /// specifies the min size (also, initial size) of the memory, while the end value specifies
+    /// the max size allowed to grow.
+    ///
+    /// # Errors
+    ///
+    /// If fail to create a [`MemType`], then an error is returned.
+    ///
+    /// # Example
+    ///
+    /// ```ignore
```

Comment:
Why does this example ignore?

Reply:
The "ignore" here notifies cargo not to do doc test against this piece of code. Thanks!